cmd/syft/cli/options/registry.go

```diff
@@ -1,21 +1,30 @@
-package config
+package options
 
 import (
 	"os"
 
-	"github.com/spf13/viper"
-
+	"github.com/anchore/clio"
 	"github.com/anchore/stereoscope/pkg/image"
+	"github.com/anchore/syft/internal/redact"
 )
 
 type RegistryCredentials struct {
 	Authority string `yaml:"authority" json:"authority" mapstructure:"authority"`
-	// IMPORTANT: do not show the username in any YAML/JSON output (sensitive information)
-	Username string `yaml:"-" json:"-" mapstructure:"username"`
-	// IMPORTANT: do not show the password in any YAML/JSON output (sensitive information)
-	Password string `yaml:"-" json:"-" mapstructure:"password"`
-	// IMPORTANT: do not show the token in any YAML/JSON output (sensitive information)
-	Token string `yaml:"-" json:"-" mapstructure:"token"`
+	Username  string `yaml:"username" json:"username" mapstructure:"username"`
+	Password  string `yaml:"password" json:"password" mapstructure:"password"`
+	Token     string `yaml:"token" json:"token" mapstructure:"token"`
+}
+
+var _ clio.PostLoader = (*RegistryCredentials)(nil)
+
+func (r *RegistryCredentials) PostLoad() error {
+	// TODO ensure that the list of RegistryCredentials has PostLoad called
```

Comment:
Do not merge without validating this!

Reply:
Needed a fangs update (represented with the go mod replace directive); this is working correctly